filesystems/blob/src/main/scala/cromwell/filesystems/blob/BlobPathBuilderFactory.scala

```diff
@@ -11,10 +11,13 @@ import scala.concurrent.ExecutionContext
 import scala.concurrent.Future
 
 final case class BlobPathBuilderFactory(globalConfig: Config, instanceConfig: Config) extends PathBuilderFactory {
+  val sasToken: String = instanceConfig.getString("filesystems.blob.instance.sas-token")
+  val container: String = instanceConfig.getString("filesystems.blob.instance.store")
+  val endpoint: String = instanceConfig.getString("filesystems.blob.instance.endpoint")
+  val workspaceId: String = instanceConfig.getString("filesystems.blob.instance.workspaceId")
+  val workspaceManagerURL = globalConfig.getString("filesystems.blob.global.workspace-manager-url")
```

Comment:
If you `import net.ceedubs.ficus.Ficus._` you can make this a little nicer. Not a big difference in this case but it makes more complicated config handling easier. More of FYI than a requested change.
```
import net.ceedubs.ficus.Ficus._
val sasToken = instanceConfig.as[String]("filesystems.blob.instance.sas-token")

# Optional value with default
val maxRetries: Int = instanceConfig.as[Option[Int]]("maxRetries").getOrElse(5)
```